### ISSUE TITLE
Timestamp field and InsertOrUpdate

### ISSUE THREAD

**Issue by MaciejWierzchowski** [borisdj/EFCore.BulkExtensions#116]
Hi,

I think I'm seeing a problem when I have a timestamp column and when I use `BulkInsertOrUpdate` and  `SetOutputIdentity = true`.
From what I was able to figure out:
The `timestamp` column is added as a last column in the temp output table (Added later on, not with the `SELECT TOP 0` statement. 
While the `MERGE` operation does output the `timestamp`, but does so in the order that is alphabetical.

Basically:
``` sql
   MERGE [table]
(...)
   OUTPUT 
       INSERTED.a,
       INSERTED.b,
       INSERTED.timestamp,
       INSERTED.z INTO [tabletempzzzz]
```

and the order in the output table is like this:
`a,b,z,timestamp`

**Comment by borisdj** [comment]
What exception are you having exactly, and can you make changes to Test _InsertWithDbComputedColumnsAndOutput_ which would reproduce it.
I have trued changing
`context.BulkInsert(`
with
`context.BulkInsertOrUpdate(`
and it still passed successfully.

**Comment by borisdj** [comment]
I have tried that as well, moved property _[Timestamp]VersionChange_ before _Content_ and still passes.

**Comment by MaciejWierzchowski** [comment]
Hmm.. just cloned the repo, and the test fails:
`Message: System.Data.SqlClient.SqlException : Invalid column name 'VersionChange'.`

If I rename the `VersionChange` to `AersionChange` the test fails as well, but with different error:

`Message: System.Data.SqlClient.SqlException : Operand type clash: timestamp is incompatible with nvarchar(max)`

Any idea what's happening? I have SQL Server 2017 if that makes any difference.

**Comment by borisdj** [comment]
Glad you solved it.

**Comment by MaciejWierzchowski** [comment]
Well, I just solved the problem of the unit testing failing when I run it. not the original problem, and that is still valid. When I rename the `VersionChange` to `AersionChange` in the `TestContext`, the test fails, and this is the problem I'm seeing.

**Comment by borisdj** [comment]
Please copy here the code of this Entity with TimeStamp property and it's modelBuilder config.

**Comment by borisdj** [comment]
Fixed with version 2.3.6

**Comment by MaciejWierzchowski** [comment]
Great, thanks for quick intervention!
